homeassistant/components/mochad/switch.py

```diff
@@ -28,12 +33,16 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Set up X10 switches over a mochad controller."""
     mochad_controller = hass.data[DOMAIN]
-    devs = config.get(CONF_DEVICES)
+    devs = config[CONF_DEVICES]
```

Comment:
Note: is the `PLATFORM_SCHEMA` correct here? Shouldn't it be `vol.Required(CONF_DEVICES)`?